app/components/maps/StreetViewImage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class StreetViewImage extends React.Component {
```

Comment:
Seems we may be moving away from using these at all, but either way here is an easy, reuseable implementation